test/runtests.jl

```diff
@@ -2,9 +2,26 @@ using Survey
 using Test
 using CategoricalArrays
 
+const STAT_TOL = 1e-5
+const SE_TOL = 1e-1
+
+# Simple random sample
+apisrs = load_data("apisrs") # Load API dataset
+srs = SurveyDesign(apisrs, weights = :pw) 
+srs_boot = srs |> bootweights # Create replicate design
+# Stratified sample
+apistrat = load_data("apistrat") # Load API dataset
+strat = SurveyDesign(apistrat, strata = :stype, weights = :pw) # Create SurveyDesign
+strat_boot = strat |> bootweights # Create replicate design
+
+# One-stage cluster sample
+apiclus1 = load_data("apiclus1") # Load API dataset
+apiclus1[!, :pw] = fill(757/15,(size(apiclus1,1),)) # Correct api mistake for pw column
+dclus1 = SurveyDesign(apiclus1; clusters =  :dnum, weights = :pw) # Create SurveyDesign
+dclus1_boot = dclus1 |> bootweights # Create replicate design
+
```

Comment:
so this is where to keep global data objects?

Reply:
Yes, once and for all in runtests.jl